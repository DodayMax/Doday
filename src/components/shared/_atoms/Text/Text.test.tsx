import * as React from 'react';
import { shallow } from 'enzyme';
import { Text } from '@components';

describe('Text', () => {
  it('renders correctly without props', () => {
    const text = shallow(<Text text='test' />);

    expect(text).toMatchSnapshot();
    expect(text.find('span').contains('test')).toBe(true);
  });
});
