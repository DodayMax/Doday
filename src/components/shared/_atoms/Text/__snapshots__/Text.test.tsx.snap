// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders correctly without props 1`] = `ShallowWrapper {}`;
